packages/features/bookings/lib/handleNewBooking/test/recurring-event.test.ts

```diff
@@ -701,109 +1163,65 @@ describe("handleNewBooking", () => {
-                meetingId: "MOCK_ID",
-                meetingPassword: "MOCK_PASS",
-                meetingUrl: "http://mock-dailyvideo.example.com/meeting-1",
-              },
-              {
-                type: "google_calendar",
-                uid: "MOCKED_GOOGLE_CALENDAR_EVENT_ID",
-                meetingId: "MOCKED_GOOGLE_CALENDAR_EVENT_ID",
-                meetingPassword: "MOCK_PASSWORD",
-                meetingUrl: "https://UNUSED_URL",
-              },
-            ],
-          });
+        const createdBookings1 = await handleRecurringEventBooking(req, res);
 
-          expectBookingCreatedWebhookToHaveBeenFired({
-            booker,
-            organizer,
-            location: "integrations:daily",
-            subscriberUrl: "http://my-webhook.example.com",
-            //FIXME: File a bug - All recurring bookings seem to have the same URL. They should have same CalVideo URL which could mean that future recurring meetings would have already expired by the time they are needed.
-            videoCallUrl: `${WEBAPP_URL}/video/${createdBookings[0].uid}`,
-          });
-        }
+        const assignedUserIds1 = createdBookings1.map((booking) => booking.userId);
 
-        // expectWorkflowToBeTriggered();
+        const createdBookings2 = await handleRecurringEventBooking(req1, res1);
 
-        expectSuccessfulBookingCreationEmails({
-          booking: {
-            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-            uid: createdBookings[0].uid!,
-            urlOrigin: CAL_URL,
-          },
-          booker,
-          organizer,
-          emails,
-          iCalUID: "MOCKED_GOOGLE_CALENDAR_ICS_ID",
-          recurrence: {
-            ...recurrence,
-            count: recurringCountInRequest,
-          },
-        });
+        const assignedUserIds2 = createdBookings2.map((booking) => booking.userId);
 
-        expect(emails.get().length).toBe(2);
+        const allSameHosts = (assignedUserIds: (number | null | undefined)[]) =>
+          assignedUserIds.every((id) => id === assignedUserIds[0]);
 
-        expectSuccessfulCalendarEventCreationInCalendar(calendarMock, [
-          {
-            calendarId: "event-type-1@google-calendar.com",
-            videoCallUrl: "http://mock-dailyvideo.example.com/meeting-1",
-          },
-          {
-            calendarId: "event-type-1@google-calendar.com",
-            videoCallUrl: "http://mock-dailyvideo.example.com/meeting-1",
-          },
-          {
-            calendarId: "event-type-1@google-calendar.com",
-            videoCallUrl: "http://mock-dailyvideo.example.com/meeting-1",
-          },
-          {
-            calendarId: "event-type-1@google-calendar.com",
-            videoCallUrl: "http://mock-dailyvideo.example.com/meeting-1",
-          },
-        ]);
-      },
-      timeout
-    );
+        expect(createdBookings1[0].userId).toBe(102); // user 101 is busy on the second recurring slot
```

Comment:
user id 101 is not available on the second slot of both recurring bookings:

- checks if the two separate booked recurring events are both assigned to user id 102 
- checks if a series of bookings is assigned to the same host (all are assigned to user id 102)